roles/foreman_installer/tasks/install.yml

```diff
@@ -12,6 +12,11 @@
 - debug:
     msg: "Foreman installer version {{ foreman_installer_version_file['content'] | b64decode }}"
 
+- name: 'Set initial admin password'
+  set_fact:
+    foreman_installer_options_internal_use_only: "{{ [ '--foreman-initial-admin-password', '{{ foreman_installer_admin_password }}' ] + foreman_installer_options_internal_use_only }}"
+  when: foreman_installer_scenario != "foreman-proxy-content" && not foreman_installer_upgrade|bool
```

Comment:
```suggestion
  when:
    - foreman_installer_scenario != "foreman-proxy-content" 
    - && not foreman_installer_upgrade|bool
```

Reply:
I am locally running into this as well which is exactly why this is a draft :)